2] loto/main.py

```diff
@@ -0,0 +1,62 @@
+from random import shuffle
+
+def lotto_user():
+    """Funkce pro čísla uživatele, které si zadá na klávesnici"""
+
+    print("Zadej unikátní čísla v rozsahu 1 - 49")
+    
+    try:
+        numbers = []
+        for x in range(6):
+            number = int(input("Zadej číslo:"))
+            if 0 < number <= 49 and number not in numbers:
+                
+                numbers.append(number)
+            else:
+                print("Číslo není v rozsahu nebo není unikátní")
+                return None
+        numbers.sort()
+        return numbers
+            
+    except ValueError:
+        print("Toto není číslo!")
+        return None
```

Comment:
Takhle ne. Tahle logika je chybna. Ty obalujes v try cely ten loop a to neni ok, protoze kdyz kupr. hned v prvni iteraci zada cislo 99 tak to vyskoci s None a ty v tom listu nic nemas. Toto neni spravne reseni. Tim try nesmis obalit cely ten loop, ale musis ten try/except mit uvnitr toho loopu